backend/JiperBackend/Examples/NewOrder.json

```diff
@@ -0,0 +1,20 @@
+{
+    "paymentType": "CARD",
+    "userId": "7",
+    "senderAddress": {
+        "city": "city",
+        "street": "str",
+        "houseNr": "1354",
+        "zipCode": "6545"
+    },
+    "package": "1",
```

Comment:
I would suggest naming it to "packageId".

Reply:
Ignas said he named it "package" in frontend so I left it like that